CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ Release date: TBD
 ### Bug Fixes
  - Fixed wrong cursor for "Edit" and "Remove" in Setting page
  - Fixed an issue where "Zoom in/out" does not properly work
+ - "Allow mixed content" does not disable web-security generally
```

Comment:
Is there any other exact description? For example, ""Allow mixed content" did not disable insecure css/javascript". Alternatively, ""Allow mixed content" now allows only insecure images" in Improvements section. Any thoughts?